machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -44,8 +44,18 @@ by adding SpotMarketOptions to the MachineSet YAML file.
 
 include::modules/machineset-yaml-azure.adoc[leveloffset=+3]
 
+Azure MachineSets support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-azure.html#machineset-non-guaranteed-instance_creating-machineset-azure[Spot VMs],
+which can save you on costs compared to standard VM prices. You can
+xref:../machine_management/creating_machinesets/creating-machineset-azure.html#machineset-creating-non-guaranteed-instance_creating-machineset-azure[configure Spot VMs]
+by adding SpotVMOptions to the MachineSet YAML file.
+
 include::modules/machineset-yaml-gcp.adoc[leveloffset=+3]
 
+GCP MachineSets support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-gcp.html#machineset-non-guaranteed-instance_creating-machineset-gcp[preemptible VM instances],
+which can save you on costs compared to normal instance prices. You can
+xref:../machine_management/creating_machinesets/creating-machineset-gcp.html#machineset-creating-non-guaranteed-instance_creating-machineset-gcp[configure preemptible VM instances]
+by adding the `Preemptible` field to the MachineSet YAML file.
+
 include::modules/machineset-creating.adoc[leveloffset=+2]
 
 [id="moving-resources-to-infrastructure-machinesets"]
```

Comment:
Heading anchors should refer to the context. I.e:

[id="moving-resources-to-infrastructure-machinesets_{context}"]

Reply:
> Heading anchors should refer to the context. I.e:
> 
> [id="moving-resources-to-infrastructure-machinesets_{context}"]

@pneedle-rh, I'll look into updating these heading anchors. This is outside of the scope of this PR and I'm not sure if I would break any links if I updated the anchors.